website/content/docs/connect/security.mdx

```diff
@@ -97,3 +97,25 @@ using a local Connect proxy. This is documented in the
 
 **If non-proxy traffic can communicate with the service**, this traffic
 will not be encrypted or authorized via Connect.
+
+### Restrict Access to Envoy's Administration Interface
+
+Envoy exposes a local, unauthenticated
+[administration interface](https://www.envoyproxy.io/docs/envoy/latest/operations/admin)
+that can be used to query and modify different aspects of the proxy. The admin
+interface allows retrieving potentially sensitive information such as:
+
+* Envoy configuration.
+* TLS certificates
+* List of upstream services and endpoints
+
+In addition, the admin interface allows performing operations which affect how
+connections are routed to or from the proxy within the service mesh.
+
+It is **highly recommended** that this endpoint only be exposed on a loopback
+address (default configuration), and access restricted to a subset of users.
```

Comment:
```suggestion
We **highly recommend** only exposing the endpoint on a loopback
address (default configuration) and restricting access to a subset of users.
```
I think 'we' is OK in this because it is guidance that actually comes from HashiCorp.